crates/key-value-azure/src/store.rs

```diff
@@ -86,15 +88,16 @@ impl KeyValueAzureCosmos {
         let database_client = cosmos_client.database_client(database);
         let client = database_client.collection_client(container);
 
-        Ok(Self { client })
+        Ok(Self { client, app_id })
     }
 }
 
 #[async_trait]
 impl StoreManager for KeyValueAzureCosmos {
-    async fn get(&self, _name: &str) -> Result<Arc<dyn Store>, Error> {
+    async fn get(&self, name: &str) -> Result<Arc<dyn Store>, Error> {
         Ok(Arc::new(AzureCosmosStore {
             client: self.client.clone(),
+            partition_key: self.app_id.as_ref().map(|i| format!("{i}/{name}")),
```

Comment:
If `app_id` is None will this incorrectly leave a leading forward slash in the partition key?

Reply:
I see below that it seems like the leading slash is okay, but I'll leave the question here to double check.